sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/GraphResourcesOperationsTests.cs

```diff
@@ -0,0 +1,162 @@
+                InitializeClients();
+                resourceGroupName = Recording.GenerateAssetName(CosmosDBTestUtilities.ResourceGroupPrefix);
+                databaseAccountName = Recording.GenerateAssetName("amegraphtest");
+                await CosmosDBTestUtilities.TryRegisterResourceGroupAsync(ResourceGroupsOperations, CosmosDBTestUtilities.Location, resourceGroupName);
+            }
+        }
+
+        [TearDown]
+        public async Task CleanupResourceGroup()
+        {
+            await CleanupResourceGroupsAsync();
+        }
+
+        [TestCase]
+        public async Task GraphCRUDTests()
+        {
+            var locations = new List<Location>();
+            Location loc = new Location();
+            loc.LocationName = CosmosDBTestUtilities.Location;
+            loc.FailoverPriority = 0;
+            loc.IsZoneRedundant = false;
+            locations.Add(loc);
+
+            DatabaseAccountCreateUpdateParameters databaseAccountCreateUpdateParameters = new DatabaseAccountCreateUpdateParameters(locations);
+            databaseAccountCreateUpdateParameters.Location = CosmosDBTestUtilities.Location;
+            databaseAccountCreateUpdateParameters.Capabilities.Add(new Capability("EnableGremlin"));
+            databaseAccountCreateUpdateParameters.ConsistencyPolicy = new ConsistencyPolicy(DefaultConsistencyLevel.Eventual);
+
+            await WaitForCompletionAsync(await CosmosDBManagementClient.DatabaseAccounts.StartCreateOrUpdateAsync(resourceGroupName, databaseAccountName, databaseAccountCreateUpdateParameters));
+
+            Task<Response> taskIsDatabaseNameExists = CosmosDBManagementClient.DatabaseAccounts.CheckNameExistsAsync(databaseAccountName);
+            Response isDatabaseNameExists = taskIsDatabaseNameExists.ConfigureAwait(false).GetAwaiter().GetResult();
+            Assert.AreEqual(200, isDatabaseNameExists.Status);
+
+            GremlinDatabaseCreateUpdateParameters gremlinDatabaseCreateUpdateParameters = new GremlinDatabaseCreateUpdateParameters(new GremlinDatabaseResource(databaseName), new CreateUpdateOptions());
+            var gremlinDatabaseResponse = await WaitForCompletionAsync(await CosmosDBManagementClient.GremlinResources.StartCreateUpdateGremlinDatabaseAsync(resourceGroupName, databaseAccountName, databaseName, gremlinDatabaseCreateUpdateParameters));
+            GremlinDatabaseGetResults gremlinDatabaseGetResults = gremlinDatabaseResponse.Value;
+            Assert.NotNull(gremlinDatabaseGetResults);
+            Assert.AreEqual(databaseName, gremlinDatabaseGetResults.Name);
+
+            var gremlinDatabaseResponse1 = await WaitForCompletionAsync(await CosmosDBManagementClient.GremlinResources.StartCreateUpdateGremlinDatabaseAsync(resourceGroupName, databaseAccountName, databaseName, gremlinDatabaseCreateUpdateParameters));
+            GremlinDatabaseGetResults gremlinDatabaseGetResults1 = gremlinDatabaseResponse1.Value;
+            Assert.NotNull(gremlinDatabaseGetResults);
```

Comment:
Assert the response is not null before assert the value of the response is not null

Reply:
Updated